### PR TITLE
Implement network tests [WIP]

### PR DESCRIPTION
**Pull request by VicGrygorchyk**

### CODE REVIEW

File: tests/test_network.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+
+class TestResponseHeader:
+
+    @sync
+    @chrome_only
+    async def test_header_contains_info(self, server, isolated_page):
+        """Verify response header contains expected info."""
+        page = isolated_page
+        # server.setRoute('/empty.html', (req, res) = > {
```
Comment (VicGrygorchyk) [inline tests/test_network.py]: @Mattwmaster58 Desperately need help here: not sure how to handle this via the test server.

Comment (Mattwmaster58) [reply tests/test_network.py:108]: Sec

Comment (Mattwmaster58) [reply tests/test_network.py]: See the matrix room

File: tests/test_network.py [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+        #     res.setHeader('foo', 'bar')
+        #     res.end()
+        # })
```
Comment (Mattwmaster58) [inline tests/test_network.py]: Add this as a placeholder in the meantime:
```suggestion
        server.app.set_one_time_response(server.empty_page, b'', headers={'foo': 'bar'})
```

**Comment by qq490456661** [comment]
goods

**Pull request closed without merging**